src/MusicalScore/ScoreIO/InstrumentReader.ts

```diff
@@ -163,24 +161,98 @@ export class InstrumentReader {
           if (newPageAttr?.value === "yes") {
             currentMeasure.printNewPageXml = true;
           }
-        } else if (xmlNode.name === "note") {
+        }
+        else if (xmlNode.name === "attributes") {
```

Comment:
Please don't start the else if on a new line. The removed line shows you our style which is used througout the project ( else if starts directly after the closing bracket)
(this also applies to further lines below)

Reply:
I removed all the unnessecary newlines and spaces .